Makefile

```diff
@@ -160,7 +160,7 @@ olm-deploy-operator: oci-build oci-push
 # Deploy the operator bundle and the operator via OLM into
 # an Kubernetes cluster selected via KUBECONFIG.
 .PHONY: olm-deploy
-ifeq ($(or $(findstring openshift-logging,$(IMG)),$(findstring openshift-logging,$(BUNDLE_IMG))),openshift-logging)
+ifeq ($(oc $(findstring openshift-logging,$(IMG)),$(findstring openshift-logging,$(BUNDLE_IMG))),openshift-logging)
```

Comment:
I think this was a mistake

Reply:
This change? No, it was intentional. Is `or` a program I'm missing or something?

Reply:
@Red-GV No `or` is a makefile conditional.